### ISSUE TITLE
createdAt / updatedAt when "create" is called

### ISSUE THREAD

**Issue by kenjinsama** [balderdashy/sails-mongo#249]
Hello,

Today, i was browsing on my application when my attention was catched by theses lines:

Creation date: 2014-12-18T13:19:26.009Z
Last update: 2014-12-18T13:19:26.010Z

I use sails-mongo and the document inserted has two date (createdAt and updatedAt) that are differents (this document never has been updated).

I do not have problems actually with that, but if someone want to check if a document never has been modified it can go bad.

Thanks for the attention.

**Comment by mikedevita** [comment]
:+1: imo updatedAt should be null until an `update` is actually done..

**Comment by stherman42** [comment]
It's an idea. But it will probably lead to a fall for those who have an application in production. Best is probably to set dates like this:

```
var doc = {};
var date = new date;

doc.createdAt = date;
doc.updatedAt = date;
```

So it will works for everyone ?

**Comment by mikedevita** [comment]
Yeah I'm  sure it would. 

This works too.

**Comment by dinana** [comment]
+1

**Comment by sailsbot** [comment]
Thanks for posting, @kenjinsama.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
